Author: Andreas Tille <tille@debian.org>
Last-Changed: Mon, 19 May 2014 20:52:12 +0200
Description: Cleanup Makefile.am from non-free docs

--- a/doc/manuals/Makefile.am
+++ b/doc/manuals/Makefile.am
@@ -3,13 +3,6 @@
 manualsdir = $(pkgdatadir)/doc/manuals
 
 manuals_DATA = \
-admin.tex \
-admin.aux \
-admin.dvi \
-admin.log \
-admin.pdf \
-admin.ps \
-admin.toc \
 adminguide.tar.gz \
 domainatrix.doc \
 program.ps.gz \
